### ISSUE TITLE
Include links to sample application

### ISSUE THREAD

**Issue by surfmuggle** [dotnet/docs#15874]
It would be really helpful to have a medium complex sample application with an tutorial that creates the sample and explains design decisions and the architecture. 

I found [getting started](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/start-mvc?view=aspnetcore-3.0&tabs=visual-studio) but this not what i mean

The other samples i found seem to be archived or not exactly what i am searching
 
1. The [Music Store on github](https://github.com/aspnet/MusicStore) and [the store tutorial on docs.msft.com](https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-1) but it is archived (at least on github). 
2. The [NerdDinner Application](https://www.hanselman.com/blog/FreeASPNETMVCEBookNerdDinnercomWalkthrough.aspx) from 2009. It seems that there is no current version. Maybe the MvcMovie Sample is the successor  
 3. [MvcMovie App](https://github.com/aspnet/AspNetCore.Docs/tree/master/aspnetcore/tutorials/first-mvc-app/start-mvc/sample/2.2-completed) which currently has no readme. Also it seems that there is no tutorial for it.
 4. The [eShopOnWeb sample application](https://aka.ms/WebAppArchitecture) seems to have no explanatory documentation. It is mentioned that the sample implements the concepts of the e-book [Architect Modern Web Applications with ASP.NET Core and Azure](https://dotnet.microsoft.com/learn/aspnet/architecture) but this does not have the easy learning curve the mvc music store tutorial did. 

To sum it up

 - use the complex eShopOnWeb sample and add a tutorial that creates the application and explains the concepts
 - create an overview of the current sample applications and write something about their complexity and goal. 

Thanks
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d49bc05d-7c8e-5f89-aac5-d133225da4d5
* Version Independent ID: 2f831fc9-dfe6-15db-4ec6-00f59c2dacc2
* Content: [.NET Core Guide](https://docs.microsoft.com/en-us/dotnet/core/)
* Content Source: [docs/core/index.md](https://github.com/dotnet/docs/blob/master/docs/core/index.md)
* Product: **dotnet-core**
* GitHub Login: @richlander
* Microsoft Alias: **mairaw**

**Comment by Thraka** [comment]
@surfmuggle Hi there! Thank you for opening this issue. We're definitely looking at redoing a lot of this introductory material. We're also going to look at redoing the tutorials too. Though, I think we could have some better cross linking in these areas.

One problem we have is that .NET Core is now geared towards two main stacks: 

1. Console apps
2. Desktop apps (Windows Desktop is built-in, 3rd party libraries enable Linux/Mac Desktop like http://avaloniaui.net/ or https://github.com/qmlnet/qmlnet)

When you want something like the sample apps you listed above, you're in the ASP.NET Core domain, which is all documented over at https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-3.0

Does this help?

**Comment by surfmuggle** [comment]
Hello @Thraka sorry i raised this issue for the wrong page. 

The points above are still valid:

 * [Introduction to ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/index?view=aspnetcore-3.0) does link to a zip file. 
 * The tutorials section has basic tutorials for [razor-pages](https://docs.microsoft.com/en-us/aspnet/core/tutorials/razor-pages/?view=aspnetcore-3.0), [mvc](https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-mvc-app/?view=aspnetcore-3.0) and others. 
 * But a medium complex sample  accompanied by a tutorial seems to be missing.
